code/script.js

```diff
@@ -31,17 +35,113 @@ const showMessage = (message, sender) => {
 }
 
 // Starts here
+
+//Bot  Default Greeting
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+showMessage(`Hello there, What's your name?`, 'bot')
+}
+
+// This will display the first answer (user name)
+const handleNameInput = (event) => {
+  event.preventDefault()
+  const name = nameInput.value
+  showMessage(`My name is ${nameInput.value}`, 'user')
+  setTimeout(() => showFoodOptions(name), 1000)
+}
+
+//Bot Question 2
+const showFoodOptions = () => {
+  showMessage(`Nice to meet you ${nameInput.value}! What fika would you like?`, 'bot')
+inputWrapper.innerHTML = `
+<button id="cinnamonrollBtn">Cinnamon roll</button>
+<button id="semlaBtn">Semla</button>
+<button id="prinsesscakeBtn">Prinsess cake</button>
+`
+//Add event listener for "cinnamon roll"
+document.getElementById("cinnamonrollBtn").addEventListener("click", () => {
+  showMessage('Cinnamon roll', 'user')
+  setTimeout(showQuantity,1000)
+})
+
+//Add event listener for "semla"
+document.getElementById("semlaBtn").addEventListener("click" , () => {
+  showMessage('Semla', 'user')
+  setTimeout(showQuantity,1000)
+})
+
+//Add event listener for "prinsess cake"
+document.getElementById("prinsesscakeBtn").addEventListener("click", () => {
+   showMessage('Prinsess cake' , 'user')
+   setTimeout(showQuantity,1000)
+})
+}
+
+//Bot Question 3
+const showQuantity = () => {
+  showMessage(`Good choice! How many would you like?`, 'bot')
+  inputWrapper.innerHTML = `
+  <select id="select">
+    <option value=""> Select quantity</option>
+    <option value="one">1</option>
+    <option value="two">2</option>
+    <option value="three">3</option>
+    <option value="four">4</option>
+  </select>
+  `
+ //Add event listener for "quantity"
+document.getElementById("select").addEventListener("change", () => { 
+  showMessage(`I would like ${select.value}`, 'user')
+  setTimeout(showPrice,1000)
+})
+}
```

Comment:
Just a little little reminder about the indentation of your code, since the eventlistener section is inside of the const showQuantity.